packages/arb-node-core/staker/staker.go

```diff
@@ -247,40 +238,40 @@ func (s *Staker) Act(ctx context.Context) (*arbtransaction.ArbTransaction, error
 	}
 	if !nodesLinear {
 		logger.Warn().Msg("fork detected")
-		if effectiveStrategy == DefensiveStrategy {
-			effectiveStrategy = StakeLatestStrategy
+		if effectiveStrategy == configuration.DefensiveStrategy {
+			effectiveStrategy = configuration.StakeLatestStrategy
 		}
 		s.inactiveLastCheckedNode = nil
 	}
 	if s.bringActiveUntilNode != nil {
 		if info.LatestStakedNode.Cmp(s.bringActiveUntilNode) < 0 {
-			if effectiveStrategy == DefensiveStrategy {
-				effectiveStrategy = StakeLatestStrategy
+			if effectiveStrategy == configuration.DefensiveStrategy {
+				effectiveStrategy = configuration.StakeLatestStrategy
 			}
 		} else {
 			logger.Info().Msg("defensive validator staked past incorrect node; waiting here")
 			s.bringActiveUntilNode = nil
 		}
 		s.inactiveLastCheckedNode = nil
 	}
-	if effectiveStrategy <= DefensiveStrategy && s.inactiveLastCheckedNode != nil {
+	if !effectiveStrategy.IsActive() && s.inactiveLastCheckedNode != nil {
```

Comment:
I think this condition accidentally changed from including DefensiveStrategy to not.